components/calendar/calendar.js

```diff
@@ -551,6 +596,25 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		});
 	}
 
+	willUpdate(changedProperties) {
+		super.willUpdate(changedProperties);
+
+		if (changedProperties.get('_shownYear') === undefined && changedProperties.get('_shownMonth') === undefined) return;
```

Comment:
Nit: this isn't 100% equivalent if you're expecting `_shownYear` or `_shownMonth` to be set to `undefined` from something else... but I don't think you are?

```suggestion
		if (!changedProperties.has('_shownYear') && !changedProperties.has('_shownMonth')) return;
```

Reply:
That's correct - I'd never expect anything to set these to `undefined`, but they would be `undefined` when the component initializes. So this check was preventing the event from being dispatched when they are changed from `undefined` when the component first renders. It would only be dispatched thereafter. I could be convinced that we should dispatch it initially as well, but consumers might end up doing double work.

Reply:
Ah ok that makes sense. Could be worth a comment or a unit test covering that case so that any future refactoring doesn't break it.